Update fix docs (#2931)
canonical/ubuntu-pro-client: docs/howtoguides/how_to_better_visualize_fixing_multiple_cves.md

> # How better visualized result of fixing multiple CVEs

Reviewer: ```suggestion
# How to better visualize the result when fixing multiple CVEs
```

To address that, we can use a `jq` filter. The `jq` command is mainly used to parse JSON data directly on
the terminal. To know more about it, please refer to this [page](https://manpages.ubuntu.com/manpages/xenial/man1/jq.1.html)

Before proceeding, we need to guarantee that `jq` is installed in your machine. This can be achieved
by running the following command:

```bash
$ apt update & apt install jq -y
```

Now that `jq` is installed, we can properly parse the JSON data delivered from the execute API.
As an example, let's assume we want to fix those three CVEs here: **CVE-2020-28196, CVE-2020-15180**
and **CVE-2017-9233**.

We can achieve that by running the following command:

```bash
pro api u.pro.security.fix.cve.execute.v1 --data '{"cves": ["CVE-2020-28196", "CVE-2020-15180", "CVE-2017-9233"]}' \
  | jq -r '.data.attributes.cves_data.cves[] | "\(.title) (\(.description)) - \(.status)"'
```

We can see that the command output will be something that follows this structure here:

```
CVE-2020-28196 (Kerberos vulnerability) - fixed
CVE-2020-15180 (MariaDB vulnerabilities) - not-affected
CVE-2017-9233 (Coin3D vulnerability) - fixed
```

> Note that this output consists of three information details:

Reviewer: ```suggestion
Note that each entry in this output consists of three fields:
```
